app/Mail/SendMail.php

```
@@ -0,0 +1,62 @@
+{
+    use Queueable, SerializesModels;
+
+    public array $data;
+
+    /**
+     * Create a new message instance.
+     */
+    public function __construct(array $data)
+    {
+        if (empty($data['subject'])) {
+            throw new \InvalidArgumentException('Subject is required in email data');
```

PHPMD (2.15.0)

Missing class import via use statement (line '24', column '23'). (Clean Code Rules)

(MissingImport)